lib/object-path-immutable-transaction.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const _assign = require('lodash.assign');
+const objectPath = require('object-path');
+const _isPlainObject = require('lodash.isplainobject');
+
+function normalizePath(path) {
+	return Array.isArray(path) ? path : ('' + path).split('.');
```

Comment:
I'm gonna leave this as is until we get clarity on whether object-path-immutable will want to land something like this.